yocto/ci/config-dev.yaml

```diff
@@ -2,24 +2,25 @@
 # Project configuration
 # -----------------------------
 ## Defines the name for the project
-name: mender-on-verdin
+# Set this via fly .... -v name=cpu01-bringup (or to your desired target)
+#name: cpu01-bringup
 
 # Git configuration
 # -----------------
 ## Specify the git repository to work with
 git_source: git@github.com:ci4rail/ci.os.lmp.git
 ## Specify the git branch to work with
-git_branch: main
+git_branch: feature/mergeplatformdirs
```

Comment:
I guess this should be main or something releasable

Reply:
Yes. Lets do this as the last step before we merge to main

Reply:
Ok